tox.ini

```diff
@@ -23,21 +23,21 @@ commands =
         {posargs} \
         integration
 
-[testenv:check-integ-py36]
+[testenv:check-integ-py3]
 deps =
     -r{toxinidir}/requirements.txt
     PyGObject
     pytest-cov
-    pytest==3.5.1
-basepython = python3.6
+    pytest
+basepython = python3
```

Comment:
Which python version in python3? I guess it makes sense to test both against python 3.6 for RHEL 8 and python 3.7 for Fedora.

Reply:
The tox will chose whatever python3 version found in system.
We could add RHEL 8 test once its docker images came out.
Hard coded version could mess things up as Fedora is keep updating it's python 3 version,
I don't want to  change both this file and docker image every time Fedora upgrade python3.

Reply:
Fedora provides multiple python versions for testing, so we can install both python 3.7 and 3.6 and test against both:
https://developer.fedoraproject.org/tech/languages/python/multiple-pythons.html

Reply:
Do we have any issue/bug caused by difference between py36 and 37?
I still don't know hard coded version which force us to update it once a while. We are making
our life easier not worse by using CI.

Reply:
We did not test this, yet, therefore we do not know. While I agree in general that hard coding versions should be avoided, it would only be the major version and I would be in favor of testing both py36 and py37. I am fine to have tox locally by default only the current python version but the CI should test py27, py36 and py37 IMHO.